client/src/main/java/io/pravega/client/admin/TableManager.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.admin;
+
+import com.google.common.annotations.Beta;
+import io.pravega.client.ClientConfig;
+import io.pravega.client.tables.TableConfiguration;
+import java.net.URI;
+
+/**
+ * Used to create and manage Tables.
+ */
+@Beta
+public interface TableManager extends AutoCloseable {
+    /**
+     * Creates a new instance of TableManager.
+     *
+     * @param scope         The Scope string.
+     * @param controllerUri The Controller URI.
+     * @return Instance of TableManager implementation.
+     */
+    static TableManager create(String scope, URI controllerUri) {
```

Comment:
This is not following the same structure as other code under the same package. For example, in `ReaderGroupManager`, the equivalent of this call would be `withScope`. Can we make it consistent?

Reply:
This was modeled after `StreamManager.